### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Add a resource via Azure Active Directory_Attach with Azure AD_Non-text Contrast: When the "Back" control receives the keyboard focus, the color contrast ratio of the border with respect to background is less than 3:1.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/AzureStorageExplorer#3463]
GitHubTags:#C&AI_Aug2020_BMAzure Tools Storage Explorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #Azure Tools Storage Explorer; #WCAG1.4.11; #Win32; #A11ySev2; #Benchmark; #ColorContrast;#AI4D;#FTP;#STP

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen gets opened and navigate to Next button and hit enter.
5. Navigate till "Back" control using tab key.
6. Open AI4D and check the color contrast ratio of the border with respect to the background, when the "Back" control receives the keyboard focus. 

### Actual:
When the "Back" control receives the keyboard focus, the color contrast ratio of the border with respect to background is less than 3:1.
**This issue is also observed with the "Next" and "Cancel" control.**
**This issue is similar throughout the application where there are "Back", "Next" and "Cancel" controls are present.**

### Expected:
When the "Back" control receives the keyboard focus, the color contrast ratio of the border with respect to background should be greater than or equal to 3:1.

### User Impact:
Low vision users may not able to see keyboard focus on the control.

### Attachment:
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shwarh_microsoft_com/EQhZdaSPAflEnLaBZ7vE-ZYB-MthpbKBIhgLYZI7bhjtqw?e=jzQvT5

**Comment by MRayermannMSFT** [comment]
![image](https://user-images.githubusercontent.com/21112697/96175188-5fa3d380-0edf-11eb-9a48-048b3e49a7bf.png)

**Comment by 12-shweta** [comment]
Verified the bug in the version 1.16.0-dev
The issue is Fixed and does not reproduce.
Hence closing this issue.
Please find the below link for Closing screenshot
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shwarh_microsoft_com/EfFyQ_jdl-BIokFTY48gUVYBw6KRGiIGGrxCYFppbo18uQ?e=TYSefa

**Comment by fsteffi** [comment]
**GitHub Tags:**#A11yMAS;

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
